src/SFA.DAS.Data.Infrastructure/Data/ApprenticeshipRepository.cs

```diff
@@ -6,26 +6,22 @@
 
 namespace SFA.DAS.Data.Infrastructure.Data
 {
-    public class CommitmentApprenticeshipRepository : BaseRepository, ICommitmentApprenticeshipRepository
+    public class ApprenticeshipRepository : BaseRepository, ICommitmentApprenticeshipRepository
     {
-        public CommitmentApprenticeshipRepository(string connectionString) : base(connectionString)
+        public ApprenticeshipRepository(string connectionString) : base(connectionString)
         {
 
         }
 
         public async Task Create(CommitmentsApprenticeshipEvent @event)
         {
-            //TODO: Get these from the database once these tables have been created
-            var paymentStatusId = 0;
-            var agreementStatusId = 0;
-
             await WithConnection(async c =>
             {
                 var parameters = new DynamicParameters();
                 parameters.Add("@commitmentId", @event.Id, DbType.Int64);
-                parameters.Add("@paymentStatusId", paymentStatusId, DbType.Int32);
+                parameters.Add("@paymentStatus", @event.PaymentStatus, DbType.StringFixedLength);
```

Comment:
Because the database has a fixed length of 50 chars. We could have not limit but I'm doubting that enums names will be longer than 50 chars for this enum type